src/View/View.php

```diff
@@ -1259,7 +1259,7 @@ protected function _paths($plugin = null, $cached = true)
             $themePaths,
             $pluginPaths,
             $templatePaths,
-            [dirname(__DIR__) . DS . 'Template' . DS]
+            [CAKE . 'Template' . DS]
```

Comment:
Nothing comes to my mind as a foolproof way to find the path without relying on this constant. On the other hand, this whole file relies heavily on external constants. `CAKE` is defined in the same file as `DS`. Any suggestions?